# Full or relative filepath
string filepath
# Topic name
string topic
# Frame id of message
string frame_id
# Latch message
bool latch
# Publish as pcl::PointCloud2
bool as_pcl_type
---
# Result message
string result_msg
